### ISSUE TITLE
Quickstart cdnow questions

### ISSUE THREAD

**Issue by andrewng88** [CamDavidsonPilon/lifetimes#283]
I have gone through the quickstart here

https://lifetimes.readthedocs.io/en/latest/Quickstart.html#example-using-transactional-datasets

and abit confused over some of these steps and let me know if I summarise it correctly as I'm pretty new to RFM.

1) Initially, we use BG/NBD model which means we do not need the dollar/monetary value to rank our customers. In this step ,we can determine our best customers and worst customers. With respect to worst customers, we can filter by a T value and a probability value for predicted_purchases to determine 'worst' customers that can we can take action on. 

2) For the plot_calibration_purchases_vs_holdout_purchases

https://lifetimes.readthedocs.io/en/latest/Quickstart.html#example-using-transactional-datasets

Can you pls explain the chart? Don't quite understand the axis and also the lines.

3) For the Gamma-Gamma submodel,
https://lifetimes.readthedocs.io/en/latest/Quickstart.html#the-gamma-gamma-model-and-the-independence-assumption

when we calculate 'conditional_expected_average_profit' , is this the individal client's CLV or is it the 'average transaction value' or we can use interchangeably?

If it's not the same do we have to use this. Thanks

```
# refit the BG model to the summary_with_money_value dataset
bgf.fit(summary_with_money_value['frequency'], summary_with_money_value['recency'], summary_with_money_value['T'])

print(ggf.customer_lifetime_value(
    bgf, #the model to use to predict the number of future transactions
    summary_with_money_value['frequency'],
    summary_with_money_value['recency'],
    summary_with_money_value['T'],
    summary_with_money_value['monetary_value'],
    time=12, # months
    discount_rate=0.01 # monthly discount rate ~ 12.7% annually
).head(10))
```

**Comment by psygo** [comment]
I agree that the `plot_calibration_purchases_vs_holdout_purchases()` is not easy to understand at first and I will add documentation to it soon.

**Comment by miao8829** [comment]
I also need help understanding the plot of plot_calibration_purchases_vs_holdout_purchases(). Could you help explain the axis  and lines? Thank you!

**Comment by psygo** [comment]
I could go into more details about it, but here is a simple explanation that might be enough...

The x-axis represents groups of customers that made `1`, `2`, `3`, etc purchases on the training/calibration period. The y-axis are *those same group of customers* but, now, how many purchases they've made, on average, on the test/holdout period.

So, [for example](https://lifetimes.readthedocs.io/en/latest/Quickstart.html#more-model-fitting), the group of customers that made `1` purchase during calibration made about `~0.23` (on average) purchases during the testing period. That's what the *real* data says at least. The model actually offers a `0.3` prediction.

Be careful though, because this graph is not normalized at all. The group of customers who lie on the `0` and `1` groups are actually usually `> 70%` of all the customers, so, when you make a mistake there, the model is probably really bad.

**Comment by miao8829** [comment]
Thank you @psygo !
